### ISSUE TITLE
Error in example documentation for fixtures

### ISSUE THREAD

**Issue by pytestbot** [pytest-dev/pytest#636]
Originally reported by: **Travis Johnson (BitBucket: [conslo](http://bitbucket.org/conslo), GitHub: [conslo](http://github.com/conslo))**

----------------------------------------

This statement: https://bitbucket.org/hpk42/pytest/src/d91265465608bd3777ab0c882e5d68335d7472dd/doc/en/fixture.txt?at=default#cl-100
is false, the test failure provided for the example fails on the line *before* the `assert 0` for some unrelated reason. Personally I think just removing the `assert "merlinux" in msg` line and re-producing the test failure output should be good, fixing what appears to be a bug is a bit out of scope.

----------------------------------------
- Bitbucket: https://bitbucket.org/pytest-dev/pytest/issue/636

**Comment by The-Compiler** [comment]
It seems the docs have changed in the meantime - can you still reproduce this, @conslo?

**Comment by merwok** [comment]
The issue is still there:
```
>       assert "merlinux" in msg
E       TypeError: Type str doesn't support the buffer API
```

**Comment by nicoddemus** [comment]
Thanks for the report @merwok! 

It seems this issue has been fixed already in commit 43d27ec7ed753bb9b6c447c94decbdd5fdcfcf4f.

You can see the latest version [here](https://github.com/pytest-dev/pytest/blob/master/doc/en/fixture.txt). 

This will be fixed automatically when the new version comes out and the documentation is re-generated.

Thanks again!
